### ISSUE TITLE
File not found: ./snapshots/wikiscenes_corr/final/ours/opt_enc_e024Xs-0.800.pth

### ISSUE THREAD

**Issue by tudipffmgt** [tgxs002/wikiscenes#1]
Thank you for your impressive work.
I am trying to reproduce your results with the provided datasets and as a first step I wanted to do inference with your pre-trained model.
I followed the steps of the Readme but now I am stuck.
I am running the following command:
`python infer_val.py --dataset wikiscenes_corr --cfg configs/wiki_resnet50.yaml --exp final --run ours --resume e024Xs-0.800 --infer-list data/wikiscenes_val.txt --workers 4 --mask-output-dir ./logs/masks/wikiscenes_corr/final/ours/wikiscenes_val`

This is the output:
Backbone: ResNet50
Model: AE
Loading weights from:  ./models/weights/resnet50-19c8e357.pth
File not found:  ./snapshots/wikiscenes_corr/final/ours/opt_enc_e024Xs-0.800.pth

Where do I find the file opt_enc_XXX.pth?
I only downloaded the file model_enc_XXX.pth and put it into the folder.
Do you provide it or do I have to generate it myself?

**Comment by tgxs002** [comment]
Thank you for your attention on our work! The missing file is the optimizer statistics, which is not necessary in evaluation, and it may not be the direct cause of the error. On my side, multithreading causes [the problem](https://stackoverflow.com/questions/53300965/pytorch-exception-in-thread-valueerror-signal-number-32-out-of-range). Setting ``--workers 0`` should work.

**Comment by tudipffmgt** [comment]
Thank you for your quick reply. I initially started with using `--workers 0`  using the script ./launch/infer_val.py
The .log file just shows (after I waited for 10 minutes):

> OMP: Warning #80: OMP_NUM_THREADS="0": value too small.
> OMP: Info #104: OMP_NUM_THREADS value "1" will be used.

So my idea was to manually set the workers to a higher number. I am not an expert on this ;).
When I run your approach:
`python infer_val.py --dataset wikiscenes_corr --cfg configs/wiki_resnet50.yaml --exp final --run ours --resume e024Xs-0.800 --infer-list data/wikiscenes_val.txt --workers 0 --mask-output-dir ./logs/masks/wikiscenes_corr/final/ours/wikiscenes_val`
the output is:

> Warning: # of threads is only  1
> Backbone: ResNet50
> Model: AE
> Loading weights from:  ./models/weights/resnet50-19c8e357.pth
> File not found:  ./snapshots/wikiscenes_corr/final/ours/opt_enc_e024Xs-0.800.pth

I also waited for around 10 minutes but nothing happened. The program created folders in wikiscenes_val named crf and vis but they are empty.
